.mergify.yml

```diff
@@ -1,31 +1,52 @@
-queue_rules:
-  - name: default
-    conditions:
-      - status-success=Build and Test (ubuntu-latest, 2.13.8, temurin@11)
-      - status-success=Build and Test (ubuntu-latest, 2.12.16, temurin@11)
+# This file was automatically generated by sbt-typelevel-mergify using the
+# mergifyGenerate task. You should add and commit this file to
+# your git repository. It goes without saying that you shouldn't edit
+# this file by hand! Instead, if you wish to make changes, you should
+# change your sbt build configuration to revise the mergify configuration
+# to meet your needs, then regenerate this file.
 
 pull_request_rules:
-  - name: assign and label scala-steward's PRs
-    conditions:
-      - author=dwolla-oss-scala-steward[bot]
-    actions:
-      request_reviews:
-        users: [bpholt]
-      label:
-        add: [dependency-update]
-  - name: automatic update pull requests
-    conditions:
-      - author=dwolla-oss-scala-steward[bot]
-      - -conflict # skip PRs with conflicts
-      - -draft # filter-out GH draft PRs
-    actions:
-      update:
-  - name: merge scala-steward's PRs
-    conditions:
-      - author=dwolla-oss-scala-steward[bot]
-      - status-success=Build and Test (ubuntu-latest, 2.13.8, temurin@11)
-      - status-success=Build and Test (ubuntu-latest, 2.12.16, temurin@11)
-    actions:
-      queue:
-        method: squash
-        name: default
+- name: merge scala-steward's PRs
+  conditions:
+  - author=dwolla-oss-scala-steward[bot]
```

Comment:
It'd be nice if this could also merge PRs from @scala-steward (like it does with the review requests below), but sbt-typelevel-mergify doesn't support that yet.

```yaml
  - or:
     - author=dwolla-oss-scala-steward[bot]
     - author=scala-steward
```